src/web/extension.ts

```diff
@@ -12,8 +12,12 @@ function writeYamlSchemaToKestra(globalState: vscode.Memento, yamlSchema: string
 
 function downloadSchemaCommand(globalState: vscode.Memento, apiClient: ApiClient) {
     return vscode.commands.registerCommand('kestra.schema.download', async () => {
-        const kestraUrl = (await ApiClient.getKestraUrl(true) as string);
-        const url = kestraUrl.replace(/\/$/, "") + (kestraUrl === kestraBaseUrl ? "" : "/api/v1") + "/plugins/schemas/flow";
+        const kestraUrl = (await ApiClient.getKestraApiUrl(true) as string);
+        let apiPrefix = "";
+        if (kestraUrl !== kestraBaseUrl && !kestraUrl.includes("/api/v1")) {
+            apiPrefix = "/api/v1";
+        }
```

Comment:
Why is this code only in the download schema ? 
The fact that the url doesn't always need api/v1 isn't specific to the schema, but to whether or not you're typing on api.kestra.io.